karate-junit5/src/main/java/com/intuit/karate/junit5/Karate.java

```diff
@@ -68,7 +67,7 @@ public Iterator<DynamicNode> iterator() {
             DynamicNode node = DynamicContainer.dynamicContainer(testName, featureNode);
             list.add(node);
         }
-        if (list.isEmpty()) {
+        if (!suite.ignoreJunitNoScenariosAssertion && list.isEmpty()) {
```

Comment:
may I suggest a shorter and more generic `failWhenNoScenariosFound` - I feel we can fail even when not called from junit, and I don't mind defaulting to false - which I think solves your immediate use case

Reply:
I did not want to change your default behavior was the rationale for the implementation.  I am glad to see you are okay with defaulting to false and I like your suggested name.  I'll make the name change.  Thanks for the quick review!